pkg/workloadapi/security/authorization.proto

```diff
@@ -0,0 +1,107 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
```

Comment:
Shouldn't this and workload.proto be in api package ? I don't think we can treat them as intern.
( can be in a separate PR, I understand it makes changes more difficult but seems the right way )?

Reply:
the `Address` proto is defined in the same go package in both `authorization.proto` and `workload.proto`. The idea here was to move the rbac code into the security package to remove the conflict

Reply:
ok. I don't mind keeping this here - we probably should still sync ( but from here to API ), because taking a dep on istio/istio is huge.